rust-toolchain

```diff
@@ -0,0 +1 @@
+nightly-2018-04-06
```

Comment:
seem we need to upgrade this now. 

PTAL @brson

Reply:
It's in the release-2.0 branch, I think we should use the same toolchain as before.